netbox/dcim/views.py

```diff
@@ -2616,6 +2616,13 @@ class InterfaceBulkEditView(generic.BulkEditView):
     table = tables.InterfaceTable
     form = forms.InterfaceBulkEditForm
 
+    def extra_object_field_operations(self, form, obj):
+        # Add/remove tagged VLANs
+        if form.cleaned_data.get('add_tagged_vlans', None):
+            obj.tagged_vlans.add(*form.cleaned_data['add_tagged_vlans'])
+        if form.cleaned_data.get('remove_tagged_vlans', None):
+            obj.tagged_vlans.remove(*form.cleaned_data['remove_tagged_vlans'])
+
```

Comment:
We probably want to validate the interface mode for these interfaces when bulk editing, to prevent #15924 and similar.

Reply:
Yeah, I think for this it is a valid approach.